backend/src/test/java/ca/bc/gov/backendstartapi/enums/ActivityEnumTest.java

```diff
@@ -11,39 +10,18 @@ class ActivityEnumTest {
       "Start a new registration or check on existing seedlots registrations";
 
   @Test
-  @DisplayName("getPropsTest")
-  void getPropsTest() {
-    ActivityEnum seedlotRegistration = ActivityEnum.SEEDLOT_REGISTRATION;
-
-    Assertions.assertEquals("SoilMoistureField", seedlotRegistration.getIconName());
-    Assertions.assertEquals("Seedlot registration", seedlotRegistration.getTitle());
-    Assertions.assertEquals(DESCRIPTION, seedlotRegistration.getDescription());
-    Assertions.assertEquals("/seedlot-registration", seedlotRegistration.getPage());
-  }
-
-  @Test
-  @DisplayName("getByTitleTest")
-  void getByTitleTest() {
-    String title = "Seedlot registration";
-    Optional<ActivityEnum> seedlotRegistration = ActivityEnum.getByTitle(title);
-    Assertions.assertTrue(seedlotRegistration.isPresent());
-
-    ActivityEnum activityEnum = seedlotRegistration.get();
-    Assertions.assertEquals("SoilMoistureField", activityEnum.getIconName());
-    Assertions.assertEquals("Seedlot registration", activityEnum.getTitle());
-    Assertions.assertEquals(DESCRIPTION, activityEnum.getDescription());
-    Assertions.assertEquals("/seedlot-registration", activityEnum.getPage());
-
-    Optional<ActivityEnum> notExisting = ActivityEnum.getByTitle("Nothing");
-    Assertions.assertFalse(notExisting.isPresent());
-
-    ActivityEnum myEnum = ActivityEnum.valueOf("SEEDLOT_REGISTRATION");
-
-    String enumToString = "ActivityEnum.SEEDLOT_REGISTRATION("
-        + "iconName=SoilMoistureField, "
-        + "title=Seedlot registration, "
-        + "description=Start a new registration or check on existing seedlots registrations, "
-        + "page=/seedlot-registration)";
-    Assertions.assertEquals(enumToString, myEnum.toString());
+  @DisplayName("valueOfTest")
+  void fromStringTest() {
+    ActivityEnum seedlotRegistration = ActivityEnum.valueOf("SEEDLOT_REGISTRATION");
+    Assertions.assertNotNull(seedlotRegistration);
+    Assertions.assertEquals(seedlotRegistration, ActivityEnum.SEEDLOT_REGISTRATION);
+
+    ActivityEnum parentTreeOrchard = ActivityEnum.valueOf("PARENT_TREE_ORCHARD");
+    Assertions.assertNotNull(parentTreeOrchard);
+    Assertions.assertEquals(parentTreeOrchard, ActivityEnum.PARENT_TREE_ORCHARD);
+
+    ActivityEnum seedlingRequest = ActivityEnum.valueOf("SEEDLING_REQUEST");
+    Assertions.assertNotNull(seedlingRequest);
+    Assertions.assertEquals(seedlingRequest, ActivityEnum.SEEDLING_REQUEST);
```

Comment:
The SonarLint plugin also pointed that we've swapped the arguments here: expected value should come first, followed by the actual value.

This will give us confusing messages if the assertions fail.

Reply:
Yep, indeed! Thanks for catching it.